src/GRA.Domain.Service/NewsService.cs

```diff
@@ -271,5 +272,44 @@ public async Task<int> GetLatestNewsIdAsync()
 
             return lastId;
         }
+        public async Task<List<bool>> GetWithinAWeek()
```

Comment:
If you add a property to the `Domain.Model.NewsCategory` of `IsNew`, you can set that in the loop here rather than passing around a second list of booleans.